src/utils.ts

```diff
@@ -10,6 +11,26 @@ export function parseNumber(n: string): number {
 	return num;
 }
 
+// Note: this could be a good place to put any additional call parsing logic or
+// even argument parsing logic.
+// Takes in a call or an array that contains a call and recursively parses all calls.
+export function parseCalls(data: any): any {
+	if (data instanceof GenericCall){
```

Comment:
I like your idea of continuing to include the old information along with the new information. In fact, if we are able to finagle this change so that it only _adds_ information (as opposed to _replacing_ old information with new information), it will not be considered a breaking API change :rocket:

Reply:
`callIndex` now back in.